pkgs/by-name/nv/nvs/package.nix

```
@@ -0,0 +1,40 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  lib,
+}:
+buildGoModule rec {
+  pname = "nvs";
+  version = "1.10.5";
+
+  src = fetchFromGitHub {
+    owner = "y3owk1n";
+    repo = pname;
```

We try to keep separation of repo name's and package names.
```suggestion
    repo = "nvs";
```